script/cms/compare-build-hashes.js

```diff
@@ -0,0 +1,158 @@
+/* eslint-disable no-console */
+
+const crypto = require('crypto');
+const fs = require('fs');
+const { JSDOM } = require('jsdom');
+const path = require('path');
+const { runCommandSync } = require('../utils');
+
+// Mapping of file paths (relative to a build path) to an array of hashes where
+// the indices correspond to the position of the build in the list of arguments
+// for the script. For example, the hashes for a file from two builds:
+// 'a/file/path': ['abcdef', '123456']
+const fileHashes = {};
+let jsonDiffPages = [];
+
+/**
+ * Calculates MD5 hash for a file.
+ * @param {string} filePath - Path to the file
+ * @return {string} The file hash.
+ */
+const calculateFileHash = filePath => {
+  const data = fs.readFileSync(filePath, { encoding: 'utf8' });
+  const hash = crypto.createHash('md5');
+  hash.update(data);
+  return hash.digest('hex');
+};
+
+/**
+ * Gathers file hashes for all HTML files in a build directory.
+ * Maps file paths to the hashes in a global object.
+ * @params {string} dirPath - The build directory path.
+ * @params {number} buildIndex - The index of the build from the script args.
+ */
+const collectFileHashes = (dirPath, buildIndex) => {
+  fs.readdirSync(dirPath).forEach(fileName => {
```

Comment:
This function could be refactored to only loop through the file paths from the JSON diff tool. But I didn't want to remove functionality until we are certain we won't need it for all build HTML paths.